exercises/concept/faceid-2/.docs/hints.md

```diff
@@ -27,10 +27,15 @@ Consider the [`HashSet<T>.Contains()`][hash-set-contains] method.
 
 This [documentation][reference-equality] addresses the issue of comparing instances as opposed to values.
 
+## 6. Potential workaround for failing tests
```

Comment:
The number hints should directly correspond to a task. In this case, I would add the hint either to task 3 or 4 (or both), or else maybe to the General tasks lists.

Reply:
Good to know! See if the changes look ok now.